authtools/views.py

```diff
@@ -121,11 +124,11 @@ class AuthDecoratorsMixin(NeverCacheMixin, CsrfProtectMixin, SensitivePostParame
 class LoginView(AuthDecoratorsMixin, WithCurrentSiteMixin, WithNextUrlMixin, FormView):
     form_class = AuthenticationForm
     template_name = 'registration/login.html'
-    disallow_authenticated = True
+    allow_authenticated = True
```

Comment:
add a note to the changelog for this explaining why the change and how to upgrade.